### ISSUE TITLE
Special hotkeys won't save

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#190]
```
What steps will reproduce the problem?
1.edit->preferences->special hotkeys
2.set modifiers, set key
3.click ok

What is the expected output? 
hotkey saved

What do you see instead?
sets the key, but blanks the modifiers and then clicking "ok" does nothing

What version of the product are you using? On what operating system?
autokey-gtk 0.82.2. crunchbang linux 

Please provide any additional information below.
other hotkeys save but the modifiers have to be respecified after setting the 
hotkey with "press to set".



```

Original issue reported on code.google.com by `tpowe...@gmail.com` on 1 Apr 2012 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
Please run in debug mode and attach output (see Troubleshooting wiki)
```

Original comment by `cdekter` on 2 Apr 2012 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
Sorry didn't see that, here it is:

2012-04-02 09:01:19,741 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:23,485 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:26,003 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:27,154 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:27,565 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:28,135 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:28,816 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:29,470 DEBUG - service - Raw key: u'a', modifiers: [], Key: a
2012-04-02 09:01:29,470 DEBUG - service - Window visible title: u'Set Hotkey', 
Window class: 'autokey-gtk.Autokey-gtk'
2012-04-02 09:01:29,471 DEBUG - service - No phrase/script matched hotkey
2012-04-02 09:01:29,473 DEBUG - service - Input stack at end of 
handle_keypress: [u'a']
2012-04-02 09:01:30,446 DEBUG - service - Received mouse click - resetting 
buffer
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/autokey/gtkui/dialogs.py", line 73, in on_ok
    if self.valid():
  File "/usr/lib/python2.7/dist-packages/autokey/gtkui/dialogs.py", line 443, in valid
    pattern = self.targetItem.get_applicable_regex().pattern
AttributeError: 'NoneType' object has no attribute 'pattern'
2012-04-02 09:01:32,287 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:40,832 DEBUG - service - Received mouse click - resetting 
buffer
2012-04-02 09:01:45,835 DEBUG - service - Received mouse click - resetting 
buffer

```

Original comment by `tpowe...@gmail.com` on 2 Apr 2012 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
Fixed for next release (0.90.0) - thanks
```

Original comment by `cdekter` on 4 Apr 2012 at 11:41
- Changed state: **Fixed**
